tensorbay/cli/auth.py

```diff
@@ -178,12 +185,12 @@ def _status_auth(obj: Dict[str, str], config_parser: ConfigParser, is_all: bool)
             _echo_user_info(key, config_parser)
         return
 
-    _echo_user_info(obj["profile_name"], config_parser)
+    _echo_user_info(obj.profile_name, config_parser)
 
 
 def _echo_user_info(profile_name: str, config_parser: ConfigParser) -> None:
     access_key, url = read_profile(profile_name, config_parser)
-    gas_client = get_gas(access_key, url, profile_name)
+    gas_client = get_gas(access_key, url, profile_name, config_parser)
```

Comment:
Isn't `read_profile` covered by `get_gas`?

Reply:
but `access_key` and `url` is needed for `echo`, get them from `gas_client` seems also inconvenient.